util-node/getApiDocsData.js

```diff
@@ -77,46 +73,39 @@ async function getApiDocsData(releaseVersions, callback) {
             }
           );
 
-          const resultingContent = parseMarkdown();
+          const parseMarkdownPromise = parseMarkdown();
 
-          // Our Navigation Sidebar is generated from within the Markdown Parser
-          // And it has a format that matches the <NavigationItem> type
-          navigationEntry.items.push(...getNavigationEntries());
+          // eslint-disable-next-line no-console
+          parseMarkdownPromise.catch(console.error);
 
-          const currentFilePath = path.resolve(
-            currentVersionPath,
-            // Creates the file path for English version of the file
-            `${getFileName(file)}.en.md`
-          );
+          parseMarkdownPromise.then(resultingContent => {
+            // Our Navigation Sidebar is generated from within the Markdown Parser
+            // And it has a format that matches the <NavigationItem> type
+            navigationEntry.items.push(...getNavigationEntries());
 
-          // Creates the File within the File System
-          return fsPromises.writeFile(currentFilePath, resultingContent, {
-            encoding: 'utf8',
-          });
-        };
+            const currentFilePath = path.resolve(
+              currentVersionPath,
+              // Creates the file path for English version of the file
+              `${getFileName(file)}.en.md`
+            );
 
-        const finishedItemCallback = () => {
-          remainingFilesToParse -= 1;
+            fs.writeFile(currentFilePath, resultingContent, () => {
+              remainingFilesToParse -= 1;
 
-          progressBar.update({
-            remaining: remainingFilesToParse,
-            finished: `${file.name}@${releaseData.version}`,
-          });
+              progressBar.update({ remaining: remainingFilesToParse });
 
-          dCb();
+              dCb();
+            });
+          });
         };
 
         fetch(file.download_url)
           .then(response => response.text())
           .then(parseMarkdownCallback)
-          .finally(finishedItemCallback);
-      }, 8);
-
-      // Add files to be Downloaded and Parsed to Text
-      apiDocsParser.push([...markdownFiles]);
+          .catch(() => dCb());
```

Comment:
nits
```suggestion
          .catch(dCb);
```

Reply:
`catch` has an argument that is "error", but no argument should be passed to that callback, if I recall correctly, hence why I've done this way.